pkg/sr/errors.go

```diff
@@ -0,0 +1,123 @@
+package sr
+
+// Schema Registry error codes as defined in the official Confluent Schema Registry.
+// These constants allow clients to programmatically check for specific error conditions
+// when interacting with the Schema Registry API.
+//
+// The error codes are organized by HTTP status code categories:
+//   - 404xx: Not Found errors
+//   - 409: Conflict errors
+//   - 422xx: Unprocessable Entity errors
+//   - 500xx: Internal Server errors
+//
+// Reference: https://github.com/confluentinc/schema-registry/blob/master/core/src/main/java/io/confluent/kafka/schemaregistry/rest/exceptions/Errors.java
+const (
+	// HTTP 404 - Not Found errors
+	
+	// ErrCodeSubjectNotFound indicates that the specified subject does not exist.
+	ErrCodeSubjectNotFound = 40401
```

Comment:
`Err` is usually reserved for `error` type. What do you think about making these a type that implements ` Error`?